src/engine.py

```diff
@@ -16,24 +16,27 @@ def train_one_epoch(model: nn.Module, iterator: Iterable,
 
     model.train()
     for image, label in iterator:
-        image = image.to(device)
-        label = label.to(device)
+        image = image.to(device) # [BATCH_SIZE=256, RGB=3, IMAGE_SIZE=32, IMAGE_SIZE=32]
```

Comment:
is the input dim 32 for resnet typically? or is this referring to something else?

Reply:
i believe it's just for CIFAR10, normally I think the input size for resnets is 224 (i think)